opennms-base-assembly/src/main/filtered/etc/events/opennms.events.xml

```diff
@@ -909,18 +909,16 @@
     <uei>uei.opennms.org/nodes/pathOutage</uei>
     <event-label>OpenNMS-defined node event: pathOutage</event-label>
     <descr>
-                        &lt;p&gt;The state of node %parm[nodelabel]% is unknown
-                        because the critical path to the node is down.&lt;/p&gt;
-                        &lt;p&gt;This event is generated when node outage processing
-                        determines that the critical path IP address/service for
-                        this node is not responding..&lt;/p&gt;
+                        &lt;p&gt;A path outage is ongoing. The status of one or several
```

Comment:
Can you undo these wording changes? I think the original wording is better. You can append the sentence with the link to the Path Outages list but leave the rest of the text as it was.